docs/modules/ROOT/pages/subscriptions/scaling.adoc

```diff
@@ -2,44 +2,44 @@
 = Horizontal Scaling
 
 == The Problem
-Horizontal scaling of any real time system can be tricky. Especially when dealing with long lived connections such as WebSockets.
+Horizontally scaling any real time system can be tricky, especially when dealing with long lived connections such as WebSockets.
 Consider the following example, in which _Client A_ is subscribed to a certain event, that is triggered by _Client B_:
 
 image::subscriptions/diagram1.png[title="Simple subscriptions setup"]
 
 
-In the previous example, the server running the GraphQL service, will do the following:
+The server running the GraphQL service will do the following:
 
 1. Receive the _mutation_ by _Client B_.
 2. Run the Cypher Query on Neo4j.
 3. Trigger the subscription event to _Client A_.
 
-This setup works fine for a single instance of `@neo4j/graphql`. However, when trying to scale horizontally, by adding more GraphQL servers,
+This setup works fine for a single instance of a `@neo4j/graphql` server. However, when trying to scale horizontally, by adding more GraphQL servers,
 we may encounter the following situation:
 
 image::subscriptions/diagram2.png[title="Subscriptions with 2 servers"]
 
-In this case, _Client B_ is subscribed to one server, however, when _Client A_ triggers the mutation, it queries a different server.
+In this case, _Client B_ is subscribed to one server, however, when _Client A_ triggers the mutation, it may query a different server.
```

Comment:
```suggestion
In this case, _Client B_ is subscribed to one server. 
However, when _Client A_ triggers the mutation, it may query a different server.
```